src/geometry/intersectionPoints.h

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Please rename file. Choose from:
 - IntersectionPoint
 - CTiglIntersectionPoint
 - CTiglIntersectionPointList

The file names of structs and classes should always start with a capital letter (in tigl).